doc/rst/language/spec/modules.rst

```diff
@@ -576,13 +576,12 @@ not exist or is not visible in the respective module or enumerated type.
 If an ``only`` list is left empty or an ``except`` list is followed by :math:`*`
 then no symbols are made available to the scope without prefix.
 
-A type may be listed in a ``limitation clause`` for a use of a module which
-contains tertiary method definitions for that type.  Listing the type in this
-way will impact the visibility of such methods in scopes where the ``use``
-statement is visible.  These methods cannot be specified in a
-``limitation-clause`` on their own.  Methods and fields defined in the type's
-definition or secondary methods defined in the same scope as the type are
-already visible to any instance of the type, see :ref:`Method_Calls`.
+When the ``limitation-clause`` for a use of a module contains a type, the
+visibility of its tertiary methods that are defined in that module, if any, are
+affected in the same way as the visibility of the type itself.  Methods cannot
+be specified in a ``limitation-clause`` on their own.  Fields, and primary and
```

Comment:
Above:
* visibility "are affected" -> "is affected"
* I suggest "Methods cannot be" --> "Fields and methods cannot be", unless there is a good reason to keep it unchanged.